### DISCUSSION TITLE
Can I use Stencil Buffer while keeping PostProcessing disabled?

### DISCUSSION THREAD

**Discussion by IlovingCode** [google/filament#6553] [General]
Hello, 
I'm building an UI system with Filament. I need to implement Mask using Stencil Buffer, but Filament requires PostProcessing to be enabled within Stencil Buffer. And I don't need PostProcessing for the UI system.

Could you help me on this problem?
Thank you

**Comment by bejado** [comment]
Post-processing is required to use the stencil buffer, but you can disable all the other post-processing features like color grading if you don't need them.
